packages/drawer/rc/use-drawer.js

```diff
@@ -0,0 +1,81 @@
+import { useReducer } from 'react'
+
+import {
+  combineFns,
+  createActionCreator,
+  persistSynthetic,
+  pick,
+  removeUndefinedValues,
+  switchcaseF
+} from './utils.js'
+
+export const actionTypes = {
+  close: 'CLOSE',
+  open: 'OPEN',
+  reset: 'RESET'
+}
+
+const defaultState = { controlled: false, isOpen: true }
```

Comment:
changed from `open` to `isOpen` so it was more explicit and didn't clash with the `open` action